legacy/src/main/java/org/opensearch/sql/legacy/esdomain/mapping/FieldMapping.java

```diff
@@ -118,8 +118,8 @@ public String path() {
      */
     @SuppressWarnings("unchecked")
     public String type() {
-        FieldMappingMetadata metaData = typeMappings.getOrDefault(fieldName, FieldMappingMetadata.NULL);
-        if (metaData.isNull()) {
+        FieldMappingMetadata metaData = typeMappings.get(fieldName);
+        if (metaData == null) {
```

Comment:
np: wondering why this change required?

Reply:
FieldMappingMetadata.NULL this static field is removed in the upstream os library.

Reply:
got it. thanks!